features/checkout/seeing_order_summary/seeing_addresses_on_order_summary_page.feature

```diff
@@ -0,0 +1,33 @@
+@checkout
+Feature: Seeing order addresses on order summary page
+    In order to be certain about shipping and billing address
+    As a Customer
+    I want to be able to see addresses on the order summary page
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "Lannister Coat" priced at "$19.99"
+        And the store ships everywhere for free
+        And the store allows paying offline
+        And I am a logged in customer
+
+    @ui
+    Scenario: Seeing the same shipping and billing address on order summary
+        Given I have product "Lannister Coat" in the cart
+        When I specified the shipping address as "Ankh Morpork", "Frost Alley", "90210", "France" for "Jon Snow"
+        And I complete order with "Free" shipping method and "Offline" payment
+        Then I should be on the checkout summary step
+        And I should see this shipping address as shipping and billing address
```

Comment:
```
When I specify the shipping address as "Ankh Morpork", "Frost Alley", "90210", "France" for "Jon Snow"
And I choose the different billing address
And I specify the billing address as "Ankh Morpork", "Frost Alley", "90210", "France" for "Eddard Stark"
And I complete the addressing step
And I complete order with "Free" shipping method and "Offline" payment
Then I should be on the checkout summary step
And this shipping address should be used for my order # what is "this shipping address" ??
And this billing address should be used for my order
```

"this shipping/billing address" is confusing and does not show exactly what would be my final address. We should either be more specific `And the shipping address specified in addressing step should be used for my order` or direct `And "Ankh Morpork", "Frost Alley", "90210", "France" for "Jon Snow" should be used for my order shipping address`.